src/styles/blocks/details.scss

```diff
@@ -0,0 +1,222 @@
+    }
+    .second {
+      background: url(../img/content/details/details-second.png) ;
+    }
+    .third {
+      background: url(../img/content/details/details-third.png) ;
+    }
+    .fourth {
+      background: url(../img/content/details/details-fourth.png) ;
+    }
+    .fifth {
+      background: url(../img/content/details/details-fifth.png) ;
+    }
+    .sixth {
+      background: url(../img/content/details/details-sixth.png) ;
+    }
+  }
+
+  &__picture {
+    width: 130px;
+    height: 130px;
+  }
+
+  &__subtitle {
+    margin-bottom: 8px;
+    font-size: 24px;
+    font-weight: 700;
+    line-height: 34px;
+  }
+
+  &__description {
+    width: 280px;
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 22px;
+    margin-bottom: 48px;
+  }
+
+  &__button {
+    display: flex;
+    justify-content: center;
+    font-family: Poppins;
+    background: #fff;
+    border-radius: 40px;
+    padding: 14px 131px;
+    width: 278px;
+    color: $textColorBlack;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 27px;
+  }
+
+  @media (min-width: 744px) {
+    &__title {
+      font-size: 48px;
+      font-weight: 500;
+      line-height: 48px;
+    }
+
+    &__button {
+      max-width: 392px;
+      margin: 0 auto;
+    }
+
+    &__description {
+      width: 600px;
+    }
+
+    &__pictures {
+      display: flex;
+      gap: 24px;
+      justify-content: space-between;
+      margin-bottom: 16px;
+
+      .first {
+        width: $width-big-picture;
+        height: $height-pictures;
+        background: url(../img/content/details/details-first.svg);
+      }
+      .second {
+        width: $width-little-picture;
+        height: $height-pictures;
+        background: url(../img/content/details/details-second.svg) ;
+      }
+      .third {
+        background: url(../img/content/details/details-third.svg) ;
+        width: $width-big-picture;
+        height: $height-pictures;
+      }
+      .fourth {
+        width: $width-little-picture;
+        height: $height-pictures;
+        background: url(../img/content/details/details-fourth.svg) ;
+      }
+      .fifth {
+        width: $width-little-picture;
+        height: $height-pictures;
+        background: url(../img/content/details/details-fifth.svg) ;
+      }
+      .sixth {
+        width: $width-big-picture;
+        height: $height-pictures;
+        background: url(../img/content/details/details-sixth.svg) ;
+      }
+    }
+  }
+
+  @media (min-width: 1260px) {
+    &__title {
+      font-size: 64px;
+      font-weight: 500;
+      line-height: 64px;
+      letter-spacing: -1.5px;
+
+    }
+
+    &__pictures {
+      width: 1020px;
+
+      .first {
+        transition: 0.5s;
+        width: $width-desktop-big-pictures;
+        height: $height-desktop-pictures;
+        background: url(../img/content/details/d-details-first.svg);
+
+        &:hover {
+          border-radius: 10px;
+          transform: scale(1.2);
+        }
```

Comment:
You now have the same `hover` styles in this `media` rule, so it's better to set it once for a more general class (for pictures) and not repeat the same thing.